### PR TITLE
Brielle

### PR DESCRIPTION
**Pull request by whitneyjsmith**

### CODE REVIEW

File: snakedenfitness/fitness/models.py [hunk 1]
```diff
@@ -31,6 +31,10 @@ class Workout(models.Model):
     reps = models.IntegerField()
     sets = models.IntegerField()
     weight = models.IntegerField()
+<<<<<<< HEAD
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+=======
```
Comment (whitneyjsmith) [inline snakedenfitness/fitness/models.py]: this is a merge conflict that needs fixed

Comment (briellefernelius) [reply snakedenfitness/fitness/models.py]: fixed

File: snakedenfitness/fitness/views.py [hunk 2]
```diff
@@ -44,11 +47,30 @@ def workout_form(request):
...
\ No newline at end of file
+    return render(request, 'fitness/request_trainer.html', {})
+
+def delete_workout(request, id):
+    workout = Workout.objects.get(pk=id)
+    workout.delete()
+    return redirect('fitness_home')
+    return render(request, 'fitness/delete_workout.html', {})
+
+def edit_workout(request):
```
Comment (whitneyjsmith) [inline snakedenfitness/fitness/views.py]: need to delete this function because of the duplicate function below

Comment (briellefernelius) [reply snakedenfitness/fitness/views.py]: deleted

File: snakedenfitness/fitness/models.py [hunk 3]
```diff
@@ -12,17 +12,17 @@ class TrainerProfile(models.Model):
 
 class Workout(models.Model):
     WORKOUT_TYPE = [
-        ('cardio', 'Cardio'),
-        ('strength', 'Strength'),
-        ('endurance', 'Endurance'),
+        ('Cardio', 'Cardio'),
```
Comment (whitneyjsmith) [inline snakedenfitness/fitness/models.py:15]: these need to be changed back to lowercase

File: snakedenfitness/fitness/models.py [hunk 4]
```diff
@@ -12,17 +12,17 @@ class TrainerProfile(models.Model):
...
+        ('Endurance', 'Endurance'),
     ]
 
     MUSCLE_GROUP = [
-        ('arms', 'Arms'),
-        ('shoulders', 'Shoulders'),
-        ('back', 'Back'),
-        ('core', 'Core'),
-        ('legs', 'Legs'),
+        ('Arms', 'Arms'),
```
Comment (whitneyjsmith) [inline snakedenfitness/fitness/models.py:21]: these need to be changed back to lowercase

File: snakedenfitness/fitness/forms.py [hunk 5]
```diff
@@ -1,24 +1,37 @@
...
+        ('Endurance', 'Endurance'),
     ]
-
     MUSCLE_GROUP = [
-        ('arms', 'Arms'),
-        ('shoulders', 'Shoulders'),
-        ('back', 'Back'),
-        ('core', 'Core'),
-        ('legs', 'Legs'),
+        ('Arms', 'Arms'),
```
Comment (whitneyjsmith) [inline snakedenfitness/fitness/forms.py:16]: these need to be changed back to lowercase

File: snakedenfitness/diet/models.py [hunk 6]
```diff
@@ -39,10 +39,10 @@ def __str__(self):
 
 class Meal(models.Model):
     MEAL_TYPES = [
-        ('breakfast', 'Breakfast'),
-        ('lunch', 'Lunch'),
-        ('dinner', 'Dinner'),
-        ('snack', 'Snack'),
+        ('Breakfast', 'Breakfast'),
```
Comment (whitneyjsmith) [inline snakedenfitness/diet/models.py:42]: these need to be changed back to lowercase

File: snakedenfitness/diet/forms.py [hunk 7]
```diff
@@ -1,17 +1,32 @@
 from django import forms
+from django.forms import ModelForm
+from .models import Meal
 
+class MealForm(ModelForm):
+    class Meta:
+        model = Meal
+        fields = ('meal_name', 'meal_type', 'calories', 'carbs', 'sugars', 'protein')
+        MEAL_TYPES = [
+            ('Breakfast', 'Breakfast'),
```
Comment (whitneyjsmith) [inline snakedenfitness/diet/forms.py:10]: these need to be changed back to lowercase

**Pull request merged**